cabal-install/Distribution/Client/Config.hs

```diff
@@ -852,7 +852,7 @@ commentSavedConfig = do
             IT.language        = toFlag Haskell2010,
             IT.license         = NoFlag,
             IT.sourceDirs      = Nothing,
```

Comment:
Should ` sourceDirs` have default set too?

Reply:
Yes it should, thanks.